piety-promoted-v4/modernized/14-chapter-14.adoc

```diff
@@ -130,9 +133,14 @@ So I desire we may all be clear in ourselves,
 and keep our children out of the fashions and customs of this world.
 And oh! that we were all of one heart and mind in these and other things;
 then would the work of the Lord go on easily, which is the sincere desire of your friend,
-HANNAH CARPENTER
 
-----------
+[.salutation]
```

Comment:
Oops! You marked Hannah Carpenter as a salutation, instead of a [.signed-section-signature] :)